walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,57 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.PriorityQueue;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    Collection<String> attendees = request.getAttendees();
+    int initialQueueCapacity = 1;
```

Comment:
What is the reason of having lines 24-27?

Reply:
I thought you couldn't specify a comparator without also specifying the initial capacity, but turns out I was looking at java 7 docs and it is actually fine in java 8.